.github/workflows/deploy-paas.yml

```diff
@@ -2,8 +2,8 @@ name: Deploy publish app to PAAS
 
 on:
   push:
-    branches:
-      - master
+  #  branches:
```

Comment:
Do we need this?

Reply:
happy to remove. i.e. run on push; regardless of branch. it was the way i cloned it.

Reply:
> 
> 
> Please make sure to put back the push: branches: master

done